set.mm

```diff
@@ -58687,6 +58687,130 @@ singleton of the first member (with no sethood assumptions on ` B ` ).
+
+
+  ${
+    $d A x y z w $.  $d R x y z w $.  $d B x y z w $.
+    $( Every (possibly proper) subclass of a class ` A ` with a founded,
+       partial-ordering, set-like relation ` R ` has a minimal element.  The
+       additional condition of partial ordering over ~ frmin enables avoiding
+       infinity.  (Contributed by Scott Fenton, 11-Feb-2022.) $)
+    frpomin $p |- ( ( ( R Fr A /\ R Po A /\ R Se A ) /\
+                      ( B C_ A /\ B =/= (/) ) ) ->
+                   E. x e. B A. y e. B -. y R x ) $=
+      ( vz vw c0 cv wbr wn wral wrex wcel wa wi breq1 syl5bi expr ad2antrr crab
+      wfr wpo wse w3a wss wne wex n0 wceq rabeq0 simprr notbid cbvralvw ralbidv
+      weq breq2 syl5bb rspcev ex syl cvv simprl simpl3 sylc seex syl2anc simpl1
+      sess2 ssrab2 sstrid syl21anc rexrab ralrab simplr simplrl simpll2 simpllr
+      fri poss simplrr potr syl13anc mp2and con3d idd ralimdva expimpd reximdva
+      jad syld pm2.61dne exlimdv impr ) CEUBZCEUCZCEUDZUEZDCUFZDHUGZBIZAIZEJZKZ
+      BDLZADMZWTFIZDNZFUHWRWSOZXFFDUIXIXHXFFWRWSXHXFWRWSXHOZOZXFGIZXGEJZGDUAZHX
+      NHUJXMKZGDLZXKXFXMGDUKXKXHXPXFPWRWSXHULZXHXPXFXEXPAXGDXEXLXBEJZKZGDLAFUPZ
+      XPXDXSBGDBGUPXCXRXAXLXBEQUMUNXTXSXOGDXTXRXMXBXGXLEUQUMUOURUSUTVARXKXNHUGZ
+      XDBXNLZAXNMZXFXKXNVBNZWOXNCUFZYAYCPXKDEUDZXHYDXKWSWQYFWRWSXHVCZWOWPWQXJVD
+      DCEVIVEXQGDXGEVFVGWOWPWQXJVHXKXNDCXMGDVJYGVKYDWOOYEYAYCABCXNVBEVSSVLYCXBX
+      GEJZYBOZADMXKXFXMYHYBAGDXLXBXGEQVMXKYIXEADXKXBDNZOZYHYBXEYBXAXGEJZXDPZBDL
+      YKYHOZXEXMYLXDBGDXLXAXGEQVNYNYMXDBDYNXADNZOZYLXDXDYPXCYLYNYOXCYLYNYOXCOZO
+      ZXCYHYLYNYOXCULYKYHYQVOYRDEUCZYOYJXHXCYHOYLPYRWSWPYSYKWSYHYQWRWSXHYJVPTYK
+      WPYHYQWOWPWQXJYJVQTDCEVTVEYNYOXCVCXKYJYHYQVRYKXHYHYQWRWSXHYJWATDXAXBXGEWB
+      WCWDSWEYPXDWFWJWGRWHWIRWKWLSWMRWN $.
+
+    $( Every (possibly proper) subclass of a class ` A ` with a founded,
+       partial-ordering, set-like relation ` R ` has a minimal element.  The
+       additional condition of partial ordering over ~ frmin enables avoiding
+       infinity.  (Contributed by Scott Fenton, 11-Feb-2022.) $)
+    frpomin2 $p |- ( ( ( R Fr A /\ R Po A /\ R Se A ) /\
+                      ( B C_ A /\ B =/= (/) ) ) ->
+                   E. x e. B Pred ( R , B , x ) = (/) ) $=
+      ( vy wfr wpo wse w3a wss c0 wne wa cv wbr wn wral wrex cpred wceq frpomin
+      crab vex dfpred3 eqeq1i rabeq0 bitri rexbii sylibr ) BDFBDGBDHICBJCKLMMEN
+      ANZDOZPECQZACRCDUJSZKTZACRAEBCDUAUNULACUNUKECUBZKTULUMUOKECDUJAUCUDUEUKEC
+      UFUGUHUI $.
+  $}
+
+  ${
+    $d A y $.  $d B y $.  $d R y $.
+    $( The principle of founded induction over a partial ordering.  This
+       theorem is a version of ~ frind that does not require infinity, and can
+       be used to prove ~ wfi and ~ tfi .  (Contributed by Scott Fenton,
+       11-Feb-2022.) $)
+    frpoind $p |- ( ( ( R Fr A /\ R Po A /\ R Se A ) /\ ( B C_ A /\
+                       A. y e. A ( Pred ( R , A , y ) C_ B -> y e. B ) ) ) ->
+                   A = B ) $=
```

Comment:
Here and below: we try not to multiply the number of lines; here it would fit on two (the last two lines can be merged).